_data/task.yml

```
@@ -9,8 +9,10 @@ websites:
     img: airtable.png
     tfa:
       - sms
+      - phone
       - totp
     doc: https://support.airtable.com/hc/en-us/articles/219409917
+    exception: "Software token can be bypassed by SMS."
```

```suggestion
    exception: "SMS-capable phone required."
```
I think the suggested exception is more accurate and clearer.